### ISSUE TITLE
Code Cleanup

### ISSUE THREAD

**Issue by karanthukral** [Erik-J-D/gigo#4]
rcsget.go currently has this
```
  // for the future, let's support many
  switch true {
    case strings.Index(src, "git@") != -1:
      gitGet(src, dest)
    case strings.Index(src, "git+ssh://") != -1:
      gitGet(src, dest)
    case strings.Index(src, ".git") != -1:
      gitGet(src, dest)
  }
```
Would it better to use an if statement instead since all options call the same method? or is it okay to leave like this in case more stuff is added later?

**Comment by Erik-J-D** [comment]
Nah, the switch is fine, it's pretty clean. It doesn't actually need the 'true' though, it could just be 'switch{'. Golang's great and actually breaks out of switch statements automatically, unlike the C family, etc

**Comment by Erik-J-D** [comment]
Could replace the index with Contains(), and that would return a bool instead of needing the != -1

https://golang.org/pkg/strings/#Contains
